extension.js

```diff
@@ -103,7 +78,7 @@ function updateDecorations(activeTextEditor, gremlins, regexpWithAllChars) {
           (match[0].length > 1 ? 's' : '') +
           ' (unicode U+' +
           gremlin.hexCodePoint +
-          ') here',
+          ') here'
```

Comment:
We use Prettier to format our code, and like the trailing commas:
https://github.com/nhoizey/vscode-gremlins/blob/master/package.json#L60-L64
https://prettier.io/docs/en/options.html#trailing-commas

Didn't your IDE use Prettier when saving your edits?